src/main/java/komo/fraczek/toukparking/charge/Math.java

```diff
@@ -0,0 +1,27 @@
+package komo.fraczek.toukparking.charge;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Collections;
+import java.util.function.BiFunction;
+import java.util.function.BinaryOperator;
+import java.util.function.UnaryOperator;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Math {
+
+    //     BiFunction caluculates value of given geometric series element
+    //     first term value is hard-coded.
```

Comment:
czy ten komentarz jest potrzebny?

Reply:
Nie, można się tego dość łatwo domyśleć. To trochę taka uwaga do samego siebie, że można by to sparametryzować